storage/src/vespa/storage/persistence/splitjoinhandler.h

```diff
@@ -0,0 +1,32 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "types.h"
+#include <vespa/storageapi/message/bucketsplitting.h>
+
+namespace storage {
+
+namespace spi { struct PersistenceProvider; }
+struct PersistenceUtil;
+class BucketOwnershipNotifier;
+class RecheckBucketInfoCommand;
+
+/**
+ * Handle operations that uses changes bucket ownership operations
```

Comment:
There is something wrong with this sentence.
Nitpick: Missing '.' at the end. On the next sentence as well.

Reply:
Fixed